platform_versioned_docs/version-23.1.0/enterprise/advanced-topics/use-iam-role.mdx

```diff
@@ -13,22 +13,131 @@ AWS-based customers can configure Tower to interact with other AWS Services like
 
 ## Configure the Tower IAM policy
 
-!!! tip "Assumptions in Provided Policies"
-
-    The provided policies were designed with certain assumptions:
+:::note
+The provided policies were designed with certain assumptions:
 
-    1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
-    2. **Trust Policy**: The Role should be assumable by EC2, ECS, EKS, and only specifically-named IAM actors.
+1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
+2. **Trust Policy**: The Role should be assumable by EC2, ECS, EKS, and only specifically-named IAM actors.
```

Comment:
This needs to be reworked. The Trust Policy depends on where the Tower instance is being deployed (i.e. EC2 vs EKS).
ECS is still kinda valid as a legacy option but no longer supported so TBD on whether we mention it.

Reply:
Stripped out ECS. How do we determine whether we should keep it?